openhtf/util/data.py

```diff
@@ -84,7 +84,7 @@ def assert_records_equal_nonvolatile(first, second, volatile_fields, indent=0):
     assert_records_equal_nonvolatile(first._asdict(), second._asdict(),
                                      volatile_fields, indent)
   elif hasattr(first, '__iter__') and hasattr(second, '__iter__'):
-    for idx, (fir, sec) in enumerate(itertools.izip(first, second)):
+    for idx, (fir, sec) in enumerate(zip(first, second)):
```

Comment:
I don't understand how Python2 handles this, but
http://python-future.org/compatible_idioms.html#zip-izip
makes it seem like we should change `zip` to `izip` and add `from builtins import izip` to the top.

Reply:
from builtins import izip  didn't appear to work

Reply:
this should be fine as-is